utils/interfaces/_deserializer.py

```diff
@@ -45,7 +45,7 @@ def deserialize_http_message(path, message, data, interface, key):
         return json.loads(data)
     elif interface == "library" and key == "response" and path == "/info":
         return json.loads(data)
-    elif content_type == "application/msgpack":
+    elif content_type == "application/msgpack" or content_type == "application/msgpack, application/msgpack":
```

Comment:
This is needed due to a bug in the dotnet tracer http client. We should make a point to remove this when the fix is merged. https://github.com/DataDog/dd-trace-dotnet/pull/2492